trello_webhooks/models.py

```diff
@@ -319,6 +321,19 @@ def template(self):
         """Return full path to render template, based on event_type."""
         return 'trello_webhooks/%s.html' % self.event_type
 
+    @property
+    def attachment_content_type(self):
+        """Returns the type of attachment on this event"""
+        if self.event_type == 'addAttachmentToCard':
+            attachment_url = self.action_data.get('attachment')['url']
+            attachment_content_type = urllib.urlopen(attachment_url).info().getheader('Content-Type')
```

Comment:
Think `urllib.urlopen` does a `GET` request - which could be a problem if the attachment is large.  A `HEAD` request is preferred.